### ISSUE TITLE
Cant see placed heads.

### ISSUE THREAD

**Issue by QQTGaming** [GeyserMC/Geyser#4933]
### Describe the bug

I cant see custom heads  placed on the Ground. I cant see them, I can only see them in the GUI.
![Screenshot_20240804-170913](https://github.com/user-attachments/assets/d6706cb7-0a1c-4096-ae19-d3099aebdaaa)

### To Reproduce

1. Cant see Heads on the Ground.
2. Add Heads to the Heads File. (To display them in the GUI and prob on ground)
3. Place heads on the ground.
4. Heads looking like steve and not like a mini grass block...

### Expected behaviour

See those heads wich are placed on the ground.

### Server Version and Plugins

Velocity 3.3.0-Snapshot
Geyser for Velocity 2.4.0-b625
Purpur 1.21-2275-82ccc76

Pugins: Skulls, PlotSquared, ServerSystem, WorldEdit, Floodgate.

### Geyser Dump

dump.geysermc.org/FA84UFqSWofp2jFmNhO1Y648hUUs6jyb

### Geyser Version

Geyser for Velocity 2.4.0-b625

### Minecraft: Bedrock Edition Device/Version

1.21 Bedrock, Phone and PC

**Comment by onebeastchris** [comment]
Please try updating Geyser to the latest version. Further, mind running `/paper dumpitem` while holding the skull & sending the output you get here?

**Comment by QQTGaming** [comment]
![image](https://github.com/user-attachments/assets/c5ecde45-5004-4d93-bf78-9c5b3bbab218)

**Comment by QQTGaming** [comment]
I have Updated the Plugin to the newest Version still not working...

**Comment by QQTGaming** [comment]
![image](https://github.com/user-attachments/assets/0a80e59f-c9a5-46b0-84ee-1011741bd849)
Not even the NPC have a skin on bedrock any more

**Comment by QQTGaming** [comment]
![image](https://github.com/user-attachments/assets/748e9b7a-1a48-4ba4-88b2-5010ece07433)
Here is the same image on java

**Comment by onebeastchris** [comment]
Does this issue still occur?

**Comment by gaRt3n** [comment]
> Does this issue still occur?

Yes. Still the same. I placed some "easter eggs" and these custom heads are properly integrated into the custom-skulls.yml

**Comment by onebeastchris** [comment]
If this still occurs, please provide the `custom-skulls.yml` file that is used - I'm not able to reproduce on my end after the latest changes

**Comment by onebeastchris** [comment]
Closing due to no further response
